test/EasyRdf/Sparql/ClientTest.php

```diff
@@ -462,6 +462,82 @@ public function testAcceptHeaders()
         $this->assertContains('text/turtle', $types);
     }
 
+    /**
+     * Test for issue https://github.com/sweetyrdf/easyrdf/issues/9
```

Comment:
Please can you update this to:
https://github.com/easyrdf/easyrdf/issues/305

Reply:
Done.